python/data_extraction/content_export.py

```diff
@@ -1,11 +1,17 @@
 from data_extraction import rummager
 from data_extraction import plek
-from data_extraction.helpers import slice, dig
+from data_extraction.helpers import slice, dig, merge
 import requests
 
-def content_links_generator(page_size=1000, rummager_url=plek.find('rummager'), blacklist_document_types=[]):
-    search_results = rummager.Rummager(rummager_url).search_generator(
-        {'reject_content_store_document_type': blacklist_document_types, 'fields': ['link']}, page_size=page_size)
+
+def content_links_generator(page_size=1000,
+                            additional_search_fields = {},
+                            rummager_url=plek.find('rummager'),
+                            blacklist_document_types=[]):
+    search_hash = merge({'reject_content_store_document_type': blacklist_document_types,
```

Comment:
This isn't technically hash, we should try to keep the Rubyisms to a minimum.

Reply:
renamed all 'hash' to 'dict'